<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PIRStatefulACForm</class>
 <widget class="QWidget" name="PIRStatefulACForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="acModelLabel">
       <property name="text">
        <string>Air Conditioner Model</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="powerButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Power Off</string>
       </property>
       <property name="icon">
        <iconset resource="../PierogiResources.qrc">
         <normaloff>:/icons/on-off_icon&amp;48.png</normaloff>:/icons/on-off_icon&amp;48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="acScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="acScrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>774</width>
        <height>290</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="acParamsLabel">
         <property name="text">
          <string>Air Conditioner Settings:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="modeLabel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Mode of Operation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="modeComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="tempLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Temp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="tempComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="fanLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Fan</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fanComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="turboLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Turbo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="turboComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="swingLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Swing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="swingComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="airCleanLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Air Clean</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="airCleanComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sendButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Send Combined Command (Also Power On If Off)</string>
     </property>
     <property name="icon">
      <iconset resource="../PierogiResources.qrc">
       <normaloff>:/icons/blue_square.png</normaloff>:/icons/blue_square.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../PierogiResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>PIRStatefulACForm</sender>
   <signal>sendCommandEnabled(bool)</signal>
   <receiver>sendButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PIRStatefulACForm</sender>
   <signal>airCleanComboBoxEnabled(bool)</signal>
   <receiver>airCleanComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PIRStatefulACForm</sender>
   <signal>modeComboBoxEnabled(bool)</signal>
   <receiver>modeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PIRStatefulACForm</sender>
   <signal>swingComboBoxEnabled(bool)</signal>
   <receiver>swingComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PIRStatefulACForm</sender>
   <signal>tempComboBoxEnabled(bool)</signal>
   <receiver>tempComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PIRStatefulACForm</sender>
   <signal>turboComboBoxEnabled(bool)</signal>
   <receiver>turboComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>684</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PIRStatefulACForm</sender>
   <signal>fanComboBoxEnabled(bool)</signal>
   <receiver>fanComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>676</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>sendCommandEnabled(bool)</signal>
  <signal>powerComboBoxEnabled(bool)</signal>
  <signal>tempComboBoxEnabled(bool)</signal>
  <signal>modeComboBoxEnabled(bool)</signal>
  <signal>swingComboBoxEnabled(bool)</signal>
  <signal>turboComboBoxEnabled(bool)</signal>
  <signal>airCleanComboBoxEnabled(bool)</signal>
  <signal>fanComboBoxEnabled(bool)</signal>
 </slots>
</ui>
